examples/randomWalk.js

```diff
@@ -11,13 +11,33 @@ function stepFunc() {
     return pg.geometry.toCircle(2 * Math.PI * Math.random())
 }
 
+function evenCircleStart(numPoints, radius){
+    return pg.geometry.circlePoints(numPoints, radius);
+}
+
+function archSpiralStart(a, b, n, pointsPerCycle){
+    var startPoints = Array(n);
```

Comment:
I think this array has n*pointsPerCycle points. How about startPoints = [] and we just startPoints.push(point) below?